lib/mustache/parser.rb

```diff
@@ -333,11 +342,19 @@ def scan_tag_delimiter content, fetch, padding, pre_match_position
     alias_method :'scan_tag_=', :scan_tag_delimiter
 
 
-    def scan_tag_open_partial content, fetch, padding, pre_match_position
-      @result << [:mustache, :partial, content, offset, padding]
+    def scan_tag_parent content, fetch, padding, pre_match_position
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Unused method argument - `padding`. If it's necessary, use `_` or `_padding` as an argument name to indicate that it won't be used.](https://www.codacy.com/app/rokusu/mustache/file/1131006120/issues/source?bid=2903613&fileBranchId=2904056#l345)